demo/src/components/Sidebar.tsx

```diff
@@ -73,7 +73,7 @@ export function Sidebar({
     {
       value: "custom",
       label: "Custom",
```

Comment:
Only enabling the custom wallet option for SOL and Babylon since we don't have a custom wallet config for ETH.